{"uuid":"e69cce70-705d-4c77-9a92-aff18191acaa","historyId":"857a30d9a9d202d43c0130e1288bd5fc","fullName":"com.qa.api.gorest.tests.UpdateUserTest.updateUser_createAndThenDeleteAPITest","labels":[{"name":"package","value":"com.qa.api.gorest.tests.UpdateUserTest"},{"name":"testClass","value":"com.qa.api.gorest.tests.UpdateUserTest"},{"name":"testMethod","value":"updateUser_createAndThenDeleteAPITest"},{"name":"parentSuite","value":"Rest API Test Suite"},{"name":"suite","value":"Rest API regression Test"},{"name":"subSuite","value":"com.qa.api.gorest.tests.UpdateUserTest"},{"name":"host","value":"DESKTOP-MUPOVES"},{"name":"thread","value":"15912@DESKTOP-MUPOVES.TestNG-test=Rest API regression Test-3(23)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"updateUser_createAndThenDeleteAPITest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Siddhi307911NEW] but found [Siddhi307911NEWFAIL]","trace":"java.lang.AssertionError: expected [Siddhi307911NEW] but found [Siddhi307911NEWFAIL]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\r\n\tat org.testng.Assert.assertEquals(Assert.java:122)\r\n\tat org.testng.Assert.assertEquals(Assert.java:629)\r\n\tat org.testng.Assert.assertEquals(Assert.java:639)\r\n\tat com.qa.api.gorest.tests.UpdateUserTest.updateUser_createAndThenDeleteAPITest(UpdateUserTest.java:94)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"updateUser_createAndThenDeleteAPITest failed and screenshot taken!","source":"fd4c6278-8ae0-405f-b8c1-d5a0e0a7d020-attachment","type":"text/plain"}],"parameters":[],"start":1641990199544,"stop":1641990208417}